### PR TITLE
add quote command for random quote

### PR DESCRIPTION
**Pull request by fauzipadlaw**
get random quote from [Programming Quotes API](http://quotes.stormconsultancy.co.uk/random.json).
Make request http using axios.
im new in nodejs, sorry for my bad code hihi

### CODE REVIEW

**Comment by indmind** [comment]
thanks for your contributions :fireworks: , sorry I just read your pull request, I'll review this first before I merge

File: src/commands/quote.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        }
+
+        this.VARIABLE = {
+            unknownError: 'Whoops... something error happened!',
+            url: 'http://quotes.stormconsultancy.co.uk/random.json'
+        }
+    }
+
+    async handle() {
+        var result = this.getQuote().then(data => {
```
Comment (indmind) [inline src/commands/quote.js]: can you use await instead? and a new line before the return

File: src/commands/quote.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        })
+
+        return result
+    }
+
+    async getQuote() {
+        var axios = require('axios')
+        return axios
+            .get(this.VARIABLE.url)
+            .then(response => {
```
Comment (indmind) [inline src/commands/quote.js]: this too

File: src/commands/quote.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    async handle() {
+        var result = this.getQuote().then(data => {
+            return data
+        })
+
+        return result
+    }
+
+    async getQuote() {
+        var axios = require('axios')
```
Comment (indmind) [inline src/commands/quote.js]: use import at the top

**Comment by fauzipadlaw** [comment]
fixed in this commit 5ac213f

File: src/commands/quote.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        }
+
+        this.VARIABLE = {
+            unknownError: 'Whoops... something error happened!',
+            url: 'http://quotes.stormconsultancy.co.uk/random.json'
+        }
+    }
+
+    async handle() {
+        return await this.getQuote().then(data => {
```
Comment (indmind) [inline src/commands/quote.js]:
```js
return await this.getQuote()
```

File: src/commands/quote.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+            return data
+        });
+    }
+
+    async getQuote() {
+        return axios
```
Comment (indmind) [inline src/commands/quote.js]:
```js
try{
    return await axios.get(this.VARIABLE.url)
catch(err){
    return err.message || JSON.stringify(err)
}
```

**Comment by indmind** [comment]
reference for async/await https://javascript.info/async-await

File: src/commands/quote.js (hunk 5 above)
Comment (fauzipadlaw) [reply src/commands/quote.js]: I am sorry 😄
this code ok?
`try {
        return await axios.get(this.VARIABLE.url)
            .then(response => {

                return `_${response.data.quote}_ - **${response.data.author}**`
            })
      } catch (err) {
        return err.message || JSON.stringify(err)
      }`

Comment (indmind) [reply src/commands/quote.js]: no need to use `then` if you already using `await`

Comment (indmind) [reply src/commands/quote.js]:
```js
try{
    const response = await axios.get(this.VARIABLE.url)

    return `_${response.data.quote}_ - **${response.data.author}**`
}catch(err){
    return err.message || JSON.stringify(err)
}
```

Comment (fauzipadlaw) [reply src/commands/quote.js]: ah i see. Im so sorry for my bad

Comment (fauzipadlaw) [reply src/commands/quote.js]: thank you. i learn alot

**Comment by indmind** [comment]
last request, just run `npm run fix` and add a new line after import, and I'll merge this! :smile:

**Comment by fauzipadlaw** [comment]
Done :dancer:

**Comment by indmind** [comment]
perfect! thank you so much for your contribution! :100: :fireworks:

**Comment by fauzipadlaw** [comment]
Thanks. FInally! :tada:

**Comment by indmind** [comment]
ack 3b54ffc

**Pull request merged**